### PR TITLE
CIP-0015: add test vectors 

### PR DESCRIPTION
**Pull request by disassembler**
Updates CIP to match implementation. It's not the vote pubkey that signed, but the entire vote payload.

### CODE REVIEW

**Comment by SebastienGllmt** [comment]
Would be great if we can also add a test vector

**Review by SebastienGllmt — CHANGES_REQUESTED** [review]
Needs to be updated for fund4

**Review by SebastienGllmt — APPROVED** [review]

File: CIP-0015/CIP-0015.md [hunk 1]
```diff
@@ -55,7 +55,7 @@ Voting key registration
 }
 ```
 
-Signing the voting public key with the staking private key
+Signing the voting registration with the staking private key. The voting registration payload is the CBOR encoded metadata including the 61284 key in binary format.
```
Comment (SebastienGllmt) [inline CIP-0015/CIP-0015.md:58]: Change this PR to avoid modifying CIP15 itself. Only the test vector

**Review by SebastienGllmt — CHANGES_REQUESTED** [review]

**Review by dcoutts — APPROVED** [review]
As @SebastienGllmt says, we need to change this PR to avoid modifying CIP15 itself, and only change the test vectors. So this is good to go once that's done.

**Review by SebastienGllmt — APPROVED** [review]

**Review by KtorZ — APPROVED** [review]
LGTM

**Pull request merged**